src/common/utf8.c

```diff
@@ -107,52 +135,69 @@ unsigned long decode_utf8(unsigned char *buf, int nbytes)
 
 int check_utf8(const char *buf, int len)
 {
-	unsigned char u[MAX_UTF8_SZ];
-	int enc_len = 0;
-	int i = 0;
-	while (1) {
-		unsigned int c = buf[i];
-		if (i >= len || c < 0x80 || (c & 0xC0) != 0x80) {
-			// the start of a new character. Process what we have
-			// in the buffer.
-			if (enc_len > 0) {
-				int re_encoded_len;
-				unsigned char re_encoded[MAX_UTF8_SZ];
-				unsigned long code = decode_utf8(u, enc_len);
-				if (code == INVALID_UTF8_CHAR) {
-					//printf("decoded to invalid utf8");
-					return i + 1;
+	/*
```

Comment:
@cyb70289 could you take this change to fix the indent? we use tabstop = 8, and shift width = 2.

Reply:
Original code in this C file is using shiftwidth=8, so I'm following it. Change only this function may lead to mixed coding style in one file which looks weird.
Ceph coding style (https://github.com/ceph/ceph/blob/master/CodingStyle) says C code conforms to linux kernel style as this file does. I'm not sure if it's still the preferred way. If not, I can help cleaning up the format of whole file, maybe with another patch.

Reply:
> C code conforms to linux kernel style as this file does.

@cyb70289 ack. thanks. i didn't realize that the C code is following a different coding style. =(